Jenkinsfile.public

```diff
@@ -58,7 +58,7 @@ pipeline {
                     steps {
                         sh 'psql -U postgres -c \'create database teststorj;\''
                         sh 'go run scripts/use-ports.go -from 1024 -to 10000 &'
-                        sh 'go test -parallel 4 -p 6 -vet=off $COVERFLAGS -timeout 12m -json -race ./... 2>&1 | tee .build/tests.json | go run ./scripts/xunit.go -out .build/tests.xml'
+                        sh 'go test -parallel 4 -p 6 -vet=off $COVERFLAGS -timeout 30m -json -race ./... 2>&1 | tee .build/tests.json | go run ./scripts/xunit.go -out .build/tests.xml'
```

Comment:
please revert the limit changes

Reply:
This only exists in a point release, not in master. I'm happy to revert them, but I dont see any reason to. The tests are passing, and this will not affect any future test runs.

Reply:
Ah, I didn't notice that this was for the point release... yeah, then it doesn't matter.